canonical-web-and-design/hiring.canonical.com
# Add change user feature for debugging

## Done
Add dropdown to test data for different hiring leads 
## QA
- Start database and run the project
```
docker-compose up -d
dotrun
```
- Go to http://localhost:8501/stages/application-review
- Check data changes when differents hiring leads are selected in the dropdown

 ## Issue
Fixes #126
